app/move/controllers/view.js

```diff
@@ -14,6 +15,21 @@ module.exports = function view(req, res) {
     cancellationReason === 'other'
       ? cancellationComments
       : req.t(`fields::cancellation_reason.items.${cancellationReason}.label`)
+
+  const urls = {
+    update: {},
+  }
+  updateSteps.forEach(step => {
+    const url = Object.keys(step)[0].replace(/^\//, '')
+    const key = url.replace(/-/g, '_')
+    urls.update[key] = `/move/${move.id}/edit/${url}`
+    // some assessment categories aren't suffixed with _information
+    if (key.endsWith('_information')) {
```

Comment:
This will likely work for now, but thinking about how the prison journeys work where some make use of the assessment controller on a route that doesn't end with `_information` may start to break this pattern. Something to keep in mind for when you tackle those though, not now.

Reply:
I think we're better off normalising the keys that we use to refer to these things by.

ie. `risk` or `risk_information` but not both

It is happenstance that it's possible to do this at the moment. And as I've been bleating, a centralised way to get urls pretty please ;)